app/features/feedback/README.md

```diff
@@ -14,15 +15,15 @@
 ### Workflow
 
 1. In the project builder, navigate to your workflow, and find the feedback section in the bottom right.
-1. Start creating a new rule by clicking the button.
-1. Define a unique id for the rule, check the boxes for whether you want feedback to be shown on success, failure or both. Where enabled, a default message must be defined.
-1. Select a strategy for your rule. See the Strategies section below for more information.
-1. Define any additional options required by the chosen strategy, and save the new rule.
+2. Start creating a new rule by clicking the button.
```

Comment:
These numbers are calculated by the Markdown parser. The original code was the markdown for a numbered list.